docs/policy.md

```diff
@@ -0,0 +1,169 @@
+# Policy file explained
+
+Say we generate a policy file for a project with the following dependency tree:
+```
+app.js
+└─┬ some-package
+  └─┬ entropoetry
+    └── bn.js
+```
+
+`./lavamoat/node/policy.json`:
+```
+{
+  "resources": {
+    "some-package": {
+      "globals": {
+        "Buffer.from": true
+      },
+      "packages": {
+        "some-package>entropoetry": true
+      }
+    },
+    "some-package>entropoetry": {
+      "builtin": {
+        "assert": true,
+        "buffer.Buffer": true,
+        "zlib": true
+      },
+      "globals": {
+        "console": true,
+        "process.exitCode": "write",
+      },
+      "packages": {
+        "some-package>entropoetry>bn.js": true
+      }
+    },
+    "some-package>entropoetry>bn.js": {
+      "builtin": {
+        "buffer.Buffer": true
+      },
+      "globals": {
+        "Buffer": true
+      }
+    }
+  }
+}
+```
+
+## Fields
+### resources 
+
+All packages in your dependency graph accessible via `require()`.  
+Note that each resource is not represented just by the name of the package, but by a full dependency tree path to it. LavaMoat picks the shortest dependency path for each entry in resources to prevent a malicious package crafted to be represented with the same name but not fetched from npm to hijack permissions.
+
+
+### packages and builtin
+
+All packages accessible by the dependency. In this example, `some-package` has access to `entropoetry`.This means that `some-package` can `require('entropoetry')`.
+
+`entropoetry` can also `require('assert')` - a built-in module of Node.js.
+
+
+### globals
+ All platform APIs and global variables accessible by the dependency. In this example,`entropoetry` has access to `console` and it can write to process.exitCode. Global access is read-only, unless defined by the `write` value.
```

Comment:
Instead of

```
In this example,`entropoetry` has access to `console` and it can write to process.exitCode. Global access is read-only, unless defined by the `write` value.
```

how about

```
A value of `true` grants readonly access to a global; write access can be enabled by setting the value to a `"write"` instead. (In the example above, `entropoetry` has access to `console` and can write to `process.exitCode`.)